ui/src/pages/backoffice/components/OrderList/OrderListEntry.js

```diff
@@ -30,8 +30,14 @@ export default class OrderListEntry extends Component {
             {order.metadata.vendor.name}
           </Link>
         </Item.Description>
+        {order.metadata.grand_total.currency !== order.metadata.grand_total_main_currency.currency && (
+          <Item.Description>
+            <label>grand total</label> {formatPrice(order.metadata.grand_total)}
+          </Item.Description>
+        )}
         <Item.Description>
-          <label>total</label> {formatPrice(order.metadata.grand_total)}
+          <label>grand total</label>{' '}
```

Comment:
`{' '}` is this prettier thing or is it intended?

Reply:
@zzacharo this is prettier, we can use `&nbsp;` if you prefer

Reply:
No but I think is fine to remove it, no?

Reply:
total139.41 CHF(126.74 €)

@zzacharo it will look like this then

Reply:
Then you can use something like : <label>`total ${formatPrice(...)}`</label>. It should do the trick no?